modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -1044,18 +1051,17 @@ RETURN duration('P2.5W') AS theDuration
 
 ======
 
-.+duration+
+.Extract the month based components of a `DURATION` value
```

Comment:
```suggestion
.Get the total months of a `DURATION` value
```
Check if it applies to more instances.

Reply:
had to revisit that one because it's not only about months, but month-based components as defined in the table